code/VersionedGridFieldOrderableRows.php

```diff
@@ -366,19 +366,16 @@ protected function reorderItems($list, array $values, array $order)
                 DB::query($liveQuery);
             }
         }
-        $this->invalidateCache($list->dataClass());
+        $this->invalidateCache($list);
     }
 
     /**
-     * Invalidate cache if the cache-include module is installed
-     * @param $className
+     * Invalidate cache if cache include is
+     * @param $list
      */
-    protected function invalidateCache($className)
-    {
-        $class = singleton($className);
-        if ($class->has_extension('Heyday\CacheInclude\SilverStripe\InvalidationExtension')) {
-            $dataobject = $class::get()->first();
-            $dataobject->onAfterWrite();
+    protected function invalidateCache($list) {

```

Comment:
This method needs to be renamed and the comment updated, since it's no longer specific to caching (it's just calling a generic hook).